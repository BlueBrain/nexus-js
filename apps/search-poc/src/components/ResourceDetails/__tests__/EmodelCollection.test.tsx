import EModelCollectionDetails  from '../EModelCollection';
import React from "react";
import { shallow } from "enzyme";



it('EModelCollectionDetails renders correctly', () => {
    const props = {
        resource: {
            emodels: [{
                id: "id",
                self: "self_url",
                name: "name",
                description: "description"
            }]
        }
    };


    // const result = shallow(<EModelDetails {...props} />);
    const result = shallow(<EModelCollectionDetails {...props} />);
    expect(result).toMatchSnapshot();
});