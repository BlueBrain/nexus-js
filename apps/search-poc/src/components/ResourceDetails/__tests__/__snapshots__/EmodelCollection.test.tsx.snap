// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EModelCollectionDetails renders correctly 1`] = `
<div
  className="emodel-collection-details"
>
  <Table
    bordered={true}
    childrenColumnName="children"
    className="small-table"
    columns={
      Array [
        Object {
          "key": "name",
          "render": [Function],
          "title": "Name",
        },
        Object {
          "dataIndex": "description",
          "key": "description",
          "title": "Description",
        },
      ]
    }
    dataSource={
      Array [
        Object {
          "description": "description",
          "id": "id",
          "name": "name",
          "self": "self_url",
        },
      ]
    }
    indentSize={20}
    loading={false}
    locale={Object {}}
    pagination={false}
    rowKey="id"
    showHeader={true}
    size="middle"
    sortDirections={
      Array [
        "ascend",
        "descend",
      ]
    }
    title={[Function]}
    useFixedHeader={false}
  />
</div>
`;
